client/src/components/teams/SubTeams.vue

```diff
@@ -20,37 +15,28 @@
     <v-col cols="12">
       <v-list v-model:opened="state.open">
         <v-list-group value="Users">
-          <template v-for="team in state.subteams" :key="team.teamName">
-            <v-list-group sub-group no-action :value="team.teamName">
+          <template v-for="subteam in subteamViews" :key="subteam.name">
+            <v-list-group sub-group no-action :value="subteam.subteamId">
               <template v-slot:activator="{ props }">
-                <!--avatar-->
                 <v-list-item v-bind="props">
                   <v-list-item-content>
-                    <!--team name-->
+                    <!--Subteam Info-->
                     <div class="d-flex align-center">
-                      {{ team.teamName }}
+                      {{ subteam.name }}
                       <span class="km-text rounded-lg ml-auto text-no-wrap pa-1 bg-success">
-                        {{ team.totalKM }}
+                        {{ subteam.totalKm }}
                       </span>
-
-                      <v-icon
-                        class="ml-5"
-                        icon="mdi mdi-pencil"
-                        @click="openEditSubteamDialog(team.teamId)"
-                        size="16px"
-                        id="pointer-cursor"
-                      />
+                      <!--Edit Subteam Pencil Icon-->
+                      <v-icon class="ml-5" icon="mdi mdi-pencil" @click="openEditSubteamDialog(subteam.subteamId)"
+                        size="16px" id="pointer-cursor" />
```

Comment:
I shouldnt see this pencil icon if im not a team admin - editing is only for team admin users